### PR TITLE
Create latex-md.sh

### PR DESCRIPTION
**Pull request by Clay-Mysten**

### CODE REVIEW

File: doc/utils/latex-md.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# Prereqs: Pandoc installed and in PATH - https://pandoc.org/
+# Usage: convert a directory of files from LaTex to Markdown
+# Run `latex-md.sh` from the directory containing the files
+# Output appears in the same directory; `mv *.md` as needed
+shopt -s nullglob
```
Comment (huitseeker) [inline doc/utils/latex-md.sh:6]: I wouldn't mind a 
```bash
set -e
set -o pipefail
```
as future-proofing.

File: doc/utils/latex-md.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# Prereqs: Pandoc installed and in PATH - https://pandoc.org/
```
Comment (huitseeker) [inline doc/utils/latex-md.sh:2]: Consider something like
```bash
command -v pandoc >/dev/null 2>&1 || { echo "pandoc (https://pandoc.org/) is not installed or missing from PATH, exiting."; return 1; }
```
instead?

Comment (Clay-Mysten) [reply doc/utils/latex-md.sh:2]: Great suggestion. Done.

File: doc/utils/latex-md.sh (hunk 1 above)
Comment (Clay-Mysten) [reply doc/utils/latex-md.sh:7]: Added. Please take another look.

**Review by huitseeker — APPROVED** [review]
🙏

**Pull request merged**